### ISSUE TITLE
[subviews] mission pack save harmless error

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#608]
When loading it, it triggers a chapter screen and then loads. All is fine, but the console has an error:
```
[GUIScript/ERROR]: Attribute Error:                                                                     
[Python/ERROR]: SystemError: Objects/longobject.c:998: bad argument to internal function                
[Python/ERROR]: Traceback (most recent call last):                                                      
[Python/ERROR]:   File "../gemrb/GUIScripts/bg1/LoadScreen.py", line 58, in <lambda>                    
[Python/ERROR]:     GemRB.SetTimer(lambda: LoadScreen.Close(), 500, 0)                                  
[Python/ERROR]:   File "../gemrb/GUIScripts/GUIClasses.py", line 242, in Close                          
[Python/ERROR]:     RemoveView(self, False)                                                             
[Python/ERROR]: AttributeError: Invalid view                                                            
[GameScript]: MoveBetweenAreas: Abdel to ar1000 [3048.831] face: 4 
```

**Comment by bradallred** [comment]
it is because we call `DestroyAllWindows` when we load a game which force closes all the windows including the loadscreen. We get away with this because windows stick around until another one is opened or closed and that is what is happening here. The chapter text window is causing the loadscreen to be destroyed before we are done presenting it.

**Comment by bradallred** [comment]
i think to come up with a good fix i would need a save that can demonstrate the problem. fiddling around with where `DestroyAllWindows` goes would likely just cause the chapter text window to be closed.

the better solution is probably to reach 100 percent sooner so that we can ignite the skull and close the window ourselves. An even better fix would probably be to reorder something. Feels like whatever script is causing the chapter text ought to not be able to run until after we have hit 100. Its really hard to say without a save to step put breakpoints and step though things.

**Comment by lynxlynxlynx** [comment]
Your bg1 copy doesn't come with the mission pack save? It's basically the shortcut to start TotSC immediately with a fully decked out party.

Your suggestion sounds good. We can try just changing the ctor of GameControl to set DialogFlags to DF_FREEZE_SCRIPTS instead of 0 and then manually unpause on 100%.

**Comment by bradallred** [comment]
I'll look for that save. its from GoG so I assume it has all the extras.

**Comment by lynxlynxlynx** [comment]
I tried my approach and it didn't work. Unpausing right after 100 is still too soon. But at least there doesn't appear to be any contention with the load screen, since the loading of the first area is fine (progress 100, often you even see the party in the temple) and then the script starts another area move and the textscreen. Ingame moves don't fire up the load screen, but it reaches 100 again before the error is printed anyway. 

So unless it requires a few more ticks for cleanup, I think we're where we started.
